tests/cli_spec.rb

```diff
@@ -6,7 +6,23 @@
   it 'runs a single spec' do
     run_command "#{Dir.pwd}/sass-spec.rb --command "\
                 "'#{Dir.pwd}/tests/sass_stub' "\
-                "#{Dir.pwd}/tests/fixtures/colors"
+                "#{Dir.pwd}/tests/fixtures/basic"
     expect(last_command_started).to be_successfully_executed
   end
+
+  it 'should not run todo specs by default' do
+    run_command "#{Dir.pwd}/sass-spec.rb --command "\
+                "'#{Dir.pwd}/tests/sass_stub' "\
+                "#{Dir.pwd}/tests/fixtures/todo"
```

Comment:
These `run_command` invocations have a lot of boilerplate... consider factoring out a helper function.

Reply:
Completely agree! I've already got a PR going for the next test, so I'll make a helper function there.